watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -1883,10 +1881,33 @@ def calculate_scaling_factors(self):
         for v in self.recovery_vol_phase.values():
             iscale.set_scaling_factor(v, 1)
         if iscale.get_scaling_factor(self.radius_pore) is None:
-            iscale.set_scaling_factor(self.radius_pore, 10 / value(self.radius_pore))
+            iscale.set_scaling_factor(self.radius_pore, 1 / value(self.radius_pore))
         if iscale.get_scaling_factor(self.membrane_thickness_effective) is None:
-            iscale.set_scaling_factor(self.membrane_thickness_effective, 1e7)
+            iscale.set_scaling_factor(
+                self.membrane_thickness_effective,
+                1 / self.membrane_thickness_effective.value,
```

Comment:
Generally, it's better to go with `value(object)` over `object.value` (after importing from pyomo.environ) since the current approach would work with variables but fail with expressions, whereas using `value(object)` should work across the board.